accio-main/src/main/java/io/accio/main/web/CacheResource.java

```diff
@@ -66,7 +66,7 @@ public void reloadAsync(@Suspended AsyncResponse asyncResponse)
 
     @GET
     @Path("info/{catalogName}/{schemaName}/{tableName}")
-    public void getTaskInfo(
+    public void getTaskInfos(
```

Comment:
this only get one TaskInfo back
```suggestion
    public void getTaskInfo(
```